src/main/webapp/js/user-page-loader.js

```diff
@@ -57,6 +58,19 @@ function setPageTitle() {
            const messageForm = document.getElementById('message-form');
            messageForm.action = '/messages?recipient=' + parameterUsername;
            messageForm.classList.remove('hidden');
+
+           /** Using 34 because @codestudents.com is 17 characters long
```

Comment:
So you're capping the username at 17? I don't quite get this logic.

Reply:
Ideally I'd be able to check and see if it's a 2-user page... Right now, the direct messaging page link is 2 user's pages link pushed together. EG: ktwebb@codeustudents.comrluo@codeustudents.com

I'd rather figure out a more explanatory way to figure out it's 2 people, but for now, if there's 2 people, the user field in the link has to be greater than 34 characters. Basically, it's something I'll have to work on, but this is a temporary fix.